client/src/svg/navigation.ts

```diff
@@ -5,8 +5,8 @@ import styles from './style.module.scss';
 export const allocate: SvgImageConfig = {
   className: styles.allocate,
   markup:
-    '<path stroke="#CDD1CD" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M8.688 13.268h2.28M19.053 13.268h2.28M17.734 9.751h3.6M8.688 9.751h3.6"/><path stroke="#CDD1CD" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M17 9.752a2 2 0 1 1-4 0 2 2 0 0 1 4 0Z" clip-rule="evenodd"/><path stroke="#CDD1CD" stroke-width="1.5" d="M4.5 6.75a2 2 0 0 1 2-2h17a2 2 0 0 1 2 2v8.5a2 2 0 0 1-2 2h-17a2 2 0 0 1-2-2v-8.5ZM4.5 22.117c0-1.031.895-1.867 2-1.867h17c1.105 0 2 .836 2 1.867v3.266c0 1.031-.895 1.867-2 1.867h-17c-1.105 0-2-.836-2-1.867v-3.266Z"/><path stroke="#CDD1CD" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M20.543 23.516h1.593M7.938 23.516l5.761-.016"/>',
-  viewBox: '0 0 32 32',
+    '<g clip-path="url(#a)" stroke="#CDD1CD" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"><path d="M8.124 9.66 4.666 6.048a2.05 2.05 0 0 1 2.898-2.897l.565.566.565-.566a2.048 2.048 0 1 1 2.897 2.897L8.124 9.66ZM5.313 19.035a1.563 1.563 0 1 0 0-3.125 1.563 1.563 0 0 0 0 3.125ZM11.563 19.035a1.563 1.563 0 1 0 0-3.125 1.563 1.563 0 0 0 0 3.125Z"/><path d="m.625 5.912 1.942 6.573a1.25 1.25 0 0 0 1.207.927h9.016a1.25 1.25 0 0 0 1.21-.927l2.698-10.128A1.875 1.875 0 0 1 18.51.965h.865"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20v20H0z"/></clipPath></defs>',
```

Comment:
`url` and `id` properties need to be changed to some unique values, eg. `{nameOfTheSvg}-a`, here `allocate-a`.

Having two different SVGs with elements with the same `id`/`url` can lead to them being rendered wrong on some of the browsers in some weird cases (e.g. colors mixed up). I remember clearly chansing the case of that happening in Firefox only.